tests/acceptance.suite.dist.yml

```diff
@@ -37,5 +37,5 @@ modules:
             admin email: 'admin@mydomain.com'      # email Id of the Admin
             language: 'English (United Kingdom)'   # Language in which you want the Application to be Installed
         AcceptanceHelper:
-            repo_folder: '/home/travis/build/redCOMPONENT-COM/redCORE/' # Path to the Extension to be installed via Install from folder
+            repo_folder: '/home/travis/build/redCOMPONENT-COM/redCORE/.dist/redCORE' # Path to the Extension to be installed via Install from folder
```

Comment:
yes, and you are right with your comment above, I will make it part of the test preparation with robo, so it won't fail in local
my point about making this is exactly that: to test the distributed package and not the repository "as is", because in the end we distribute a package and not a repository

I'll make the suggested changes, thanks for that :)